examples/lib/core_test.rego

```diff
@@ -18,4 +18,4 @@ test_has_field_pos {
 test_missing_field {
     input := {"kind": "test"}
     not has_field(input, "abc")
-}
+}
```

Comment:
nit: this should be reverted so there is a newline at the end of the file

Reply:
ok, changed it